tests/python/relax/test_transform_simplify_norm.py

```diff
@@ -146,7 +146,82 @@ def expected(
             R.output(out, gv1)
         return out, gv1
 
-    _check(before, expected)
+    _check(before, expected, "eval")
+
+
+def test_batch_norm_training():
+    @R.function
+    def before(
+        x: R.Tensor((1, 64, 112, 112), "float32"),
+        gamma: R.Tensor((64,), "float32"),
+        beta: R.Tensor((64,), "float32"),
+        moving_mean: R.Tensor((64,), "float32"),
+        moving_var: R.Tensor((64,), "float32"),
+    ):
+        with R.dataflow():
+            bn = R.nn.batch_norm(
+                x,
+                gamma,
+                beta,
+                moving_mean,
+                moving_var,
+                axis=1,
+                epsilon=1e-5,
+                center=True,
+                scale=True,
+                momentum=0.1,
+            )
+            gv0 = bn[0]
+            gv1 = bn[1]
+            gv2 = bn[2]
+            R.output(gv0, gv1, gv2)
+        return gv0, gv1, gv2
+
+    @R.function
+    def expected(
+        x: R.Tensor((1, 64, 112, 112), dtype="float32"),
+        gamma: R.Tensor((64,), dtype="float32"),
+        beta: R.Tensor((64,), dtype="float32"),
+        moving_mean: R.Tensor((64,), dtype="float32"),
+        moving_var: R.Tensor((64,), dtype="float32"),
+    ) -> R.Tuple(
+        R.Tensor((1, 64, 112, 112), dtype="float32"),
+        R.Tensor((64,), dtype="float32"),
+        R.Tensor((64,), dtype="float32"),
+    ):
+        with R.dataflow():
+            lv: R.Tensor((64,), dtype="float32") = R.mean(x, axis=[0, 2, 3], keepdims=False)
+            lv1: R.Tensor((1, 64, 1, 1), dtype="float32") = R.expand_dims(lv, axis=[0, 2, 3])
+            lv2: R.Tensor((1, 64, 112, 112), dtype="float32") = R.subtract(x, lv1)
+            lv3: R.Tensor((64,), dtype="float32") = R.variance(x, axis=[0, 2, 3], keepdims=False)
+            lv4: R.Tensor((1, 64, 1, 1), dtype="float32") = R.expand_dims(lv3, axis=[0, 2, 3])
+            lv5: R.Tensor((1, 64, 1, 1), dtype="float32") = R.add(
+                lv4, R.const(9.9999997473787516e-06, "float32")
```

Comment:
You can write the constants simpler, like 1e-5, 0.9, etc. That will make script easier to read, and assert_structural will not complain error